tests/Orchestration/Base.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+namespace Utopia\Tests;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use Utopia\Orchestration\Exceptions\TimeoutException;
+use Utopia\Orchestration\Orchestration;
+
+abstract class Base extends TestCase
+{
+    /**
+     * @return Orchestration
+     */
+    abstract static protected function getOrchestration(): Orchestration;
+
+    public function setUp(): void
+    {
+
+    }
+
+    public function tearDown(): void
+    {
+
+    }
+
+    public function testPullImage(): void {
+        /**
+         * Test for Success
+         */
+        $response = static::getOrchestration()->pull('appwrite/runtime-for-php:8.0');
+        
+        $this->assertEquals(true, $response);
+
+        /**
+         * Test for Failure
+         */
+
+        $testFailed = false;
+
+        try {
```

Comment:
Instead of a try-catch, we can define the exception we expect to be thrown:
`$this->expectException(\Exception::class)`

I think it's more elegant and accurate, so we can check for any future exceptions like this.
[here's an example from the database tests](https://github.com/utopia-php/database/blob/main/tests/Database/Base.php#L152-L153)